client/app/cloud/project/compute/infrastructure/diagram/cloud-project-compute-infrastructure-diagram.controller.js

```diff
@@ -95,7 +96,7 @@
             this.volumes = null;
 
             // Hide highlighted-element on change state
-            this.$scope.$on("$stateChangeStart", () => {
+            this.$transitions.onStart({ from: "iaas.pci-project.compute.infrastructure.diagram" }, () => {
```

Comment:
@frenautvh Same here. I only execute this function when right before leaving this state.